deploy/gk-deploy

```diff
@@ -138,16 +141,22 @@ assign() {
 check_pods() {
   local all_ready=false
   s=0
+  prestatus=""
   while [[ "${all_ready}" != "true" ]]; do
     if [[ ${s} -ge ${WAIT} ]]; then
       output "Timed out waiting for pods matching '${1}'."
       abort
     fi
     sleep 2
     ((s+=2))
-    pods=$(${CLI} get pod --no-headers --show-all --selector="${1}")
-    debug "Checking status of pods matching '${1}':"
-    debug "$pods"
+    pods=$(${CLI} get pod --no-headers --show-all --selector="${1}" 2>&1)
+    if [[ "${prestatus}" != "${pods}" ]]; then
+      debug "Checking status of pods matching '${1}':"
+      debug "$pods"
+      prestatus="${pods}"
+    else
+      debug "dotit"
```

Comment:
Awesome. Maybe I thought too much like having it as a functionality ;P, I didn't realize we can do this.